src/test/resources/feature/checkout2-2.feature

```diff
@@ -0,0 +1,40 @@
+Feature: complete order with new address
```

Comment:
If checkout2-2.feature and checkout2-1.feature files contains scenarios with the same area of testing so move it to common feature file and rename it file

Reply:
Done